modules/build/src/main/scala/scala/build/preprocessing/mdsandbox/MdRunner.scala

```diff
@@ -0,0 +1,30 @@
+package scala.build.preprocessing.mdsandbox
```

Comment:
```suggestion
package scala.build.internal
```
I'd change the package to internal, to put it next to the script `CustomCodeWrapper` (and adjust the file location accordingly)